Formula/dbxml.rb

```diff
@@ -34,18 +32,22 @@ def install
     ENV.cxx11
 
     inreplace "dbxml/configure" do |s|
-      s.gsub! "lib/libdb-*.la | sed -e 's\/.*db-\\\(.*\\\).la", "lib/libdb-*.a | sed -e 's/.*db-\\(.*\\).a"
-      s.gsub! "lib/libdb-*.la", "lib/libdb-*.a"
-      s.gsub! "libz.a", "libz.dylib"
+      s.gsub! %r{=`ls ("\$with_berkeleydb"/lib)/libdb-\*\.la \| sed -e 's/\.\*db-\\\(\.\*\\\)\.la/},
+              "=`find \\1 -name #{shared_library("libdb-*")} -maxdepth 1 ! -type l " \
+              "| sed -e 's/#{shared_library(".*db-\\(.*\\)")}/"
```

Comment:
I changed `ls` to `find` to avoid symlinked libraries as the configure fails if there is more than one version (e.g. 18 vs 18.1). Side effect of upstream only using static libraries and our manual override to dynamic.